airbyte-integrations/connectors/source-facebook-marketing/integration_tests/invalid_config.json

```diff
@@ -1,6 +1,5 @@
 {
   "start_date": "2023-04-01T00:00:00Z",
   "account_id": "account",
-  "access_token": "wrong_token",
```

Comment:
Why you delete access token?

Reply:
Why you decided it is deleted? Look next row.

Reply:
My bad. Sorry!